AccountingSystem/Agent/NetworkAgent.py

```diff
@@ -5,34 +5,35 @@
 from datetime import datetime
 
 from DIRAC import S_OK, S_ERROR, gLogger
-from DIRAC.Core.Base.AgentModule                  import AgentModule
-from DIRAC.AccountingSystem.Client.Types.Network  import Network
-from DIRAC.ConfigurationSystem.Client.Config      import gConfig
-from DIRAC.ConfigurationSystem.Client.CSAPI       import CSAPI
+from DIRAC.Core.Base.AgentModule import AgentModule
+from DIRAC.AccountingSystem.Client.Types.Network import Network
+from DIRAC.ConfigurationSystem.Client.Config import gConfig
+from DIRAC.ConfigurationSystem.Client.CSAPI import CSAPI
 from DIRAC.Resources.MessageQueue.MQCommunication import createConsumer
 
-__RCSID__ = "$Id: $"
+__RCSID__ = "$Id$"
 
-class NetworkAgent ( AgentModule ):
+
+class NetworkAgent (AgentModule):
   """
   AccountingSystem agent to processes messages containing perfSONAR network metrics.
   Results are stored in the accounting database.
   """
 
   BUFFER_TIMEOUT = 3600  # default number of seconds after which network accounting
-                         # objects are removed form the the temporary buffer
+  # objects are removed form the the temporary buffer
+
+  def initialize(self):
 
-  def initialize( self ):
-    
-    self.log = gLogger.getSubLogger( 'NetworkAgent' )
+    self.log = gLogger.getSubLogger('NetworkAgent')
 
     # API initialization is required to get an up-to-date configuration from the CS
     self.csAPI = CSAPI()
     self.csAPI.initialize()
 
     # temporary buffer for network accounting objects + some parameters
     self.buffer = {}  # { {addTime: datetime.now(), object: Network() }, ... }
-    self.bufferTiemout = self.am_getOption( 'BufferTimeout', NetworkAgent.BUFFER_TIMEOUT )
+    self.bufferTiemout = self.am_getOption('BufferTimeout', NetworkAgent.BUFFER_TIMEOUT)
```

Comment:
Do you want to fix the spelling in the variable name?

Reply:
Done.